rocketchat-core/src/main/java/com/rocketchat/core/callback/RoomCallback.java

```diff
@@ -18,4 +19,7 @@
         void onGetRoomMembers(Integer total, List<User> members);
     }
 
-}
+    public interface GetFilesCallback extends Callback {
+        void onGetRoomFiles(Integer total, List<Attachment> files);
```

Comment:
Maybe this could be just an `int` and avoid an autoboxing (I know that there are other parts using `Integer` already)

Reply:
In the first moment I was surprised about the Integer usage, but I was following what was done. Of course we'll change it to int! Thanks for your thoughts!

Reply:
Actually we don't really need this, do we? this is total total or size of the files list (`files.size()`)?

Reply:
A good usage of the `total` is on the current Rocket.Chat+ Android app. The total is a key that comes from the call that shows the total of files on a Room. This is total total